tests/unittests/dataset_tests.py

```diff
@@ -4833,6 +4833,203 @@ def test_select_exclude_dynamic_fields(self):
         self.assertFalse(frame.has_field("field"))
         self.assertFalse(frame.predictions.detections[0].has_field("field"))
 
+    @drop_datasets
+    def test_filter_fields(self):
```

Comment:
Thanks for the tests! Can you move these to `tests/unittests/view_tests.py`? There should already be some tests for the existing `SelectFields` syntax over there.